<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QConfig</class>
 <widget class="QWidget" name="QConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>591</width>
    <height>242</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="grpBox_globalconfig">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>271</width>
     <height>241</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Global configuration :</string>
   </property>
   <widget class="QPushButton" name="pshBut_memory">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>50</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>Right to write on memory</string>
    </property>
    <property name="text">
     <string>Enable</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_memory">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Memory protect :</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_confnum">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Set. number :</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_mastvol">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Master volume :</string>
    </property>
   </widget>
   <widget class="QSlider" name="hzSlider_mastvolume">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>140</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>Master level of the module</string>
    </property>
    <property name="maximum">
     <number>127</number>
    </property>
    <property name="value">
     <number>127</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_infos">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>231</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>These informations are not stored in files and cannot be retrieved from the module.</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_syschan">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>System channel :</string>
    </property>
   </widget>
   <widget class="QButton" name="but_syschan">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>The system channel of the module to be edited</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
    <property name="minValue" stdset="0">
     <number>1</number>
    </property>
    <property name="maxValue" stdset="0">
     <number>16</number>
    </property>
   </widget>
   <widget class="QButton" name="but_setnum">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>80</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>The current set number</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
    <property name="minValue" stdset="0">
     <number>1</number>
    </property>
    <property name="maxValue" stdset="0">
     <number>20</number>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_mastdetune">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Master detune :</string>
    </property>
   </widget>
   <widget class="QButton" name="but_mastdetune">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>110</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>Master detune of the module</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="minValue" stdset="0">
     <number>-64</number>
    </property>
    <property name="maxValue" stdset="0">
     <number>63</number>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="grpBox_setconfig">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>0</y>
     <width>271</width>
     <height>241</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Set configuration :</string>
   </property>
   <widget class="QLabel" name="lbl_combine">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Combine mode :</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pshBut_combine">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>60</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>When on, the global configuration of the last selected voice replace the instrument configuration</string>
    </property>
    <property name="text">
     <string>Enable</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="cmbBox_reception">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>90</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>Receive every or one in two MIDI orders</string>
    </property>
    <item>
     <property name="text">
      <string>All</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Even</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Odd</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="lbl_reception">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Reception mode :</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="txtEdit_name">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>The name of the set</string>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="lineWrapMode">
     <enum>QPlainTextEdit::NoWrap</enum>
    </property>
    <property name="maximumBlockCount">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_name">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Set name :</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_LFOspeed">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>LFO speed :</string>
    </property>
   </widget>
   <widget class="QButton" name="but_AMD">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>180</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>The amplitude modulation provided by the LFO</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="minValue" stdset="0">
     <number>0</number>
    </property>
    <property name="maxValue" stdset="0">
     <number>127</number>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_AMD">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>AMD :</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cmbBox_LFOwave">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>150</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>The LFO waveform for the set</string>
    </property>
    <item>
     <property name="text">
      <string>Saw</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Square</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Triangle</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>S&amp;H</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="lbl_PMD">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>PMD :</string>
    </property>
   </widget>
   <widget class="QButton" name="but_PMD">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>210</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>The pitch modulation provided by the LFO</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="minValue" stdset="0">
     <number>0</number>
    </property>
    <property name="maxValue" stdset="0">
     <number>127</number>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_LFOwave">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>LFO wave :</string>
    </property>
   </widget>
   <widget class="QButton" name="but_LFOspeed">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>120</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>The pitch modulation provided by the LFO</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="minValue" stdset="0">
     <number>0</number>
    </property>
    <property name="maxValue" stdset="0">
     <number>255</number>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QButton</class>
   <extends>QPushButton</extends>
   <header>interface/qbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
